tests/lca/imagebasedupgrade/cnf/upgrade-talm/internal/tsparams/const.go

```diff
@@ -15,6 +15,15 @@ const (
 	// IbuCguNamespace is the namespace where IBU CGUs created on target hub.
 	IbuCguNamespace = "default"
 
+	// IbguNamespace is the namespace where IBU CGUs created on target hub.
```

Comment:
Same comment as for IbuCguNamespace above.

Reply:
tnx, updated.